src/data/full-quiz.js

```diff
@@ -1,4 +1,48 @@
 const fullQuiz = [
+  {
+    Question:
+      "A variable that has been declared, but has not been assigned a value is _______________",
+    Answer: "undefined",
+    Distractor1: "unknown",
+    Distractor2: "unassigned",
+    Distractor3: "pending",
+    Explanation:
+    "A variable that has not been assigned a value is of type undefined",

```

Comment:
Proposed fix:
In JavaScript, a variable that has been declared but not assigned a value is known as undefined.